catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/metacard/AbstractBasicMetacardUpdater.java

```diff
@@ -28,7 +30,7 @@ protected AbstractBasicMetacardUpdater(String attributeName) {
     }
 
     @Override
-    public final void update(Metacard parent, Metacard child) {
+    public final void update(Metacard parent, Metacard child, Context context) {
```

Comment:
I actually don't have a big problem with making a method `final` from a conceptual perspective. I've just been bitten too many times in the past where I would want to extend a class and I did have a good reason to override a method but couldn't, or I wanted to mock a class and had to resort to PowerMock because a method or a class was `final`.

I guess it's a little like checked exceptions. They seem like a good idea in theory but in practice not so much. Imho, unless there's a really good reason for a class or method to be `final`, I usually err on the side of being flexible and not prevent clients from doing something I didn't think of when I wrote the class. But that's just me :-)